cmd/generate/config/rules/scalingo.go

```diff
@@ -0,0 +1,24 @@
+package rules
+
+import (
+	"regexp"
+
+	"github.com/zricethezav/gitleaks/v8/cmd/generate/secrets"
+	"github.com/zricethezav/gitleaks/v8/config"
+)
+
+func ScalingoAPIToken() *config.Rule {
+	// define rule
+	r := config.Rule{
+		Description: "Scalingo API token",
+		RuleID:      "scalingo-api-token",
+		Regex:       regexp.MustCompile(`tk-us-[a-zA-Z0-9-_]{48}`),
```

Comment:
I'd add a word boundary (`\b`) on both ends to avoid false-positives like this.

https://github.com/aws-samples/amazon-aurora-call-to-amazon-sagemaker-sample/blob/eb8dd7fd2cd7d2a7353da595ad7b94007c19e08a/data-ingestion-from-k8s-and-train/README.md?plain=1#L66

Reply:
Thanks for the fast review. I'm going to change that 😃